### DISCUSSION TITLE
How use node specific version?

### DISCUSSION THREAD

**Discussion by chz** [coollabsio/coolify#1208] [Q&A] [answered]
How use node specific version? Nixpacks build support define node version ? because now only node 16
<img width="324" alt="Screenshot 2023-09-08 at 15 46 49" src="https://github.com/coollabsio/coolify/assets/3233370/312d10da-fc1d-485f-84e9-86c4f44f3244">

But in my Dockerfile i use node-18

**Comment by chz** [comment, marked as answer]
Resolved.

**Reply by andrasbacsai** [reply]
Can you please state how you solved it, so others could benefit as well? :)

**Reply by chz** [reply]
> Can you please state how you solved it, so others could benefit as well? :)

Yeap. in package.json just add:

`
"engines": {
  "node": "18"
},
`

**Reply by mrvfino** [reply]
Just wanted to comment here as well to help others, because it solved my problem after hours of searching :)

I'm encountering "ReferenceError: File is not defined" because i have a component that uses the File type, and I needed to set the Node version to 20 to remedy this (as per this discussion in Vercel https://github.com/vercel/next.js/discussions/56032), but there's no GUI option in coolify.
